domain/src/main/assets/ratio_input_exploration.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
As with other explorations, please also add a state testing ratio input to the prototype exploration. Or, are you planning to do that in a later PR?

Reply:
I'll be doing that in later PR because doing that would lead to failing tests.

Reply:
Ah, makes sense & sounds good. Thanks!